package.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "@jscig/ows",
+  "version": "1.0.0",
+  "license": "AGPL-3.0",
```

```suggestion
  "license": "UNLICENSED",
```

I think it should be `UNLICENSED`.

see https://docs.npmjs.com/files/package.json#license